.travis.yml

```diff
@@ -8,8 +8,10 @@ php:
   - 5.4
 
 install:
-  - composer install --dev --prefer-source
+  - composer install --prefer-source
 
 script:
   - bin/phpunit
   - bin/psecio-parse scan src tests
+
+dist: trusty
```

Comment:
You're hard to please, aren't you? 🙄 
Is there some reason Precise is needed? It's EOL as of April.